auth-web/src/components/auth/staff/SetupAccountForm.vue

```diff
@@ -2,20 +2,55 @@
   <div>
     <div>
       <v-form ref="directorSearchForm">
-        <v-alert type="error" v-show="errorMessage">{{errorMessage}}</v-alert>
-
         <v-row>
-            <v-col class="d-flex" cols="12" xl="6">
-                <v-text-field
-                filled
-                label="Account Name"
-                v-model.trim="accountName"
-                :rules="accountNameRules"
-                persistent-hint
-                :disabled="saving"
-                />
+            <v-col class="flex pb-0" cols="12">
```

Comment:
Let's try to stick with pattern similar to the one used in 'UserProfileForm.vue' for layout here. We are only ever using a single column for this layout - it should be reflected in the markup. Single column per row will suffice for this layout.